### PR TITLE
pbTests: Synchronise branch specification in VPC & QPC

### PR DESCRIPTION
**Pull request by Willsparker**
Fixes: #1941 

VPC now specifies branches in the same way as QPC, because I think it's much cleaner. Fringe benefit, now VPC can test branches that have `/` in the name. Also just cleaned up the code a little bit. 

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [ ] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [x] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access) : This will require the job to be reconfigured, so I'll look for a suitable time to test this. Putting in draft until I can complete that.
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Comment by Willsparker** [comment]
https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/1087/ Got it to work! Coming out of draft

File: ansible/pbTestScripts/vagrantPlaybookCheck.sh [hunk 1]
```diff
@@ -240,24 +218,24 @@ startVMPlaybook()
 	sed -i -e "s/.*hosts:.*/- hosts: all/g" playbooks/AdoptOpenJDK_Unix_Playbook/main.yml
 	awk '{print}/^\[defaults\]$/{print "private_key_file = id_rsa"; print "remote_tmp = $HOME/.ansible/tmp"; print "timeout = 30"}' < ansible.cfg > ansible.cfg.tmp && mv ansible.cfg.tmp ansible.cfg
 	
-	ansible-playbook -i playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx -u vagrant -b --skip-tags adoptopenjdk,jenkins${skipFullSetup} playbooks/AdoptOpenJDK_Unix_Playbook/main.yml 2>&1 | tee $WORKSPACE/adoptopenjdkPBTests/logFiles/$folderName.$branchName.$OS.log
+	ansible-playbook -i playbooks/AdoptOpenJDK_Unix_Playbook/hosts.unx -u vagrant -b --skip-tags adoptopenjdk,jenkins${skipFullSetup} playbooks/AdoptOpenJDK_Unix_Playbook/main.yml 2>&1 | tee $WORKSPACE/adoptopenjdkPBTests/logFiles/$gitFork.$gitBranch.$OS.log
```
Comment (sxa) [inline ansible/pbTestScripts/vagrantPlaybookCheck.sh:221]: Possibly use `${gitFork}.${gitBranch} here for consistency with other changes you've made in the file

**Comment by sxa** [comment]
Running this myself to see if it works ...

**Review by karianna — APPROVED** [review]
LGTM

**Review by gdams — APPROVED** [review]

**Review by sxa — APPROVED** [review]
> Running this myself to see if it works ...

Seemed ok until I lost my connect to the server it was running on which has nothing to do with this PR ... So I'll approve

**Pull request merged**